ONSdigital/response-operations-ui
# [TBL] Call party service direct to change enrolment status

# Motivation and Context

Removes the call to backstage for changing enrolment status.

# What has changed



Changes the `change_enrolment_status` method in reporting_units_controllers.py to hit the party service directly.

# How to test?



Run the unit tests, make sure they pass.
Tail the logs for backstage, party service. Hit the `/reporting-units/<ru_ref>/change-enrolment-status` page and POST. Make sure backstage doesn't get the call, but party service does.
